src/components/Landing.js

```diff
@@ -387,99 +401,54 @@ export default class Landing extends Component {
-                      name: subitem.subSection.name,
-                      flagText: subitem.flagText,
-                      className: subitem.flagClass,
-                      text: subitem.flagText.indexOf("[DATE]") >= 0 ? (subitem.flagText.replace("[DATE]", datishFormat("", flagDateText))) : (subitem.flagText + (subitem.flagDateText ? ` (${datishFormat('',flagDateText)})`: "")),
-                      priority: subitem.priority || 100
-                    });
-                    //log alert
-                    this.writeToLog("alert flag: "+subitem.flagText, "warn", {tags: ["alert"]});
-                  }
-                });
-              } else {
-                if (subsection[1].flagText) {
+    if (!sectionFlags) {
+      return alerts;
+    }
+    //compile relevant alerts
+    for (let section in sectionFlags) {
+      for (let subsection of Object.entries(sectionFlags[section])) {
+        if (subsection[1]) {
+          if (typeof subsection[1] === "string") {
+            alerts.push(subsection[1]);
+          }
+          if (typeof subsection[1] === "object") {
+            if (Array.isArray(subsection[1])) {
+              subsection[1].forEach(subitem => {
+                //this prevents addition of duplicate alert text
+                let alertTextExist = alerts.filter(item => String(item.flagText).toLowerCase() === String(subitem.flagText).toLowerCase());
+                if (!alertTextExist.length && subitem.flagText) {
+                  let flagDateText = subitem.flagDateText ? subitem.flagDateText : "";
                   alerts.push({
-                    id: subsection[1].subSection.dataKey,
-                    name: subsection[1].subSection.name,
-                    text: subsection[1].flagText,
-                    className: subsection[1].flagClass,
-                    priority: subsection[1].priority || 100
+                    id: subitem.subSection.dataKey,
+                    name: subitem.subSection.name,
+                    flagText: subitem.flagText,
+                    className: subitem.flagClass,
+                    text: subitem.flagText.indexOf("[DATE]") >= 0 ? (subitem.flagText.replace("[DATE]", datishFormat("", flagDateText))) : (subitem.flagText + (subitem.flagDateText ? ` (${datishFormat('',flagDateText)})`: "")),
+                    priority: subitem.priority || 100
                   });
+                  //log alert
+                  this.writeToLog("alert flag: "+subitem.flagText, "warn", {tags: ["alert"]});
                 }
+              });
+            } else {
+              if (subsection[1].flagText) {
+                alerts.push({
+                  id: subsection[1].subSection.dataKey,
+                  name: subsection[1].subSection.name,
+                  text: subsection[1].flagText,
+                  className: subsection[1].flagClass,
+                  priority: subsection[1].priority || 100
+                });
               }
             }
```

Comment:
Can these `else`/`if` be flattened together into an `else if`?
```suggestion
            } else if (subsection[1].flagText) {
              alerts.push({
                id: subsection[1].subSection.dataKey,
                name: subsection[1].subSection.name,
                text: subsection[1].flagText,
                className: subsection[1].flagClass,
                priority: subsection[1].priority || 100
              });
          }
```